tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs

```diff
@@ -276,7 +276,6 @@ public void Redis_IsInstalled(string imageTag)
         [InlineData("8.2")]
         [InlineData("8.1")]
         [InlineData("8.0")]
-        [InlineData("7.4")]
```

Comment:
Same comment as above.

Reply:
please have a look on the above reply. https://github.com/microsoft/Oryx/pull/1955#discussion_r1168159697